salt/log/__init__.py

```diff
@@ -9,6 +9,8 @@
     are made to assure backwards compatibility.
 """
 
+# pylint: disable = no-name-in-module
```

Comment:
Because something in the workflow was generating an error and I was not able to identify why, mentioned it to @MKLeb too, so pragmatic solution was to disable it.
The problem occurs locally too if add a blank line for pre-commit to reject, and then it triggers on the issue.  This could be due to the old version of pylint (2.4.4) used and only with Py3.8 (form googling - false positive) hence trying various versions of pylint, but more trouble than worth dealing with.

Reply:
Ok, but what exactly triggers the no-name-in-module
@s0undt3ch We already had this conversation, about false positives with pylint 2.4.4 and py3.8, hence the pragmatic disable. If we really want to get to the bottom of it, explain why the workflow is doing a pylint check on a file that is unchanged in this PR, sha256sum verified same as branch 3006.x